### PR TITLE
sima bio added 

### PR DESCRIPTION
**Pull request by SimaRaha**
Sima pic and bio added to the CCV  website.

### CODE REVIEW

**Comment by SimaRaha** [comment]
@galenwinsor Please review my changes and let me know the next steps.

File: about/people.yml [hunk 1]
```diff
@@ -430,6 +430,17 @@ data:
...
+    name: Sima Raha
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Graphics, Software, and Data Core
+    title: Software Sustanibaility Intern 
+    github_username: SimaRaha
+    brown_directory_uuid: cf68285e-47c5-4a65-897e-187c94e4f36b
+    bio: Sima Raha is a Software Sustainability Intern at CCV. She is Class of 2025, studying Economics and Data Fluency at Brown University. Her career interests include quantitative analysis, business intelligence (BI) development, and statistical programming. She enjoys hiking, nature photography, playing frisbee, and billiards.
+    image: sima_main.jpg
+=======
```
Comment (galenwinsor) [inline about/people.yml:443]: Looks like there's still a merge conflict marker here on line 433 and 443. The `<<<<< HEAD` line means "the following changes come from the current branch, until the `=====` line." You need to delete lines 433 and 443 in order to resolve the merge conflict.

Comment (galenwinsor) [reply about/people.yml:443]: Oh, and line 472.

File: images/people/sima_main.JPG
Comment (galenwinsor) [inline images/people/sima_main.JPG:1]: Make sure the file extension matches between the hover and main photo. In this case, make the extension of this file lowercase.

**Review by galenwinsor — CHANGES_REQUESTED** [review]
Looks good Sima, just resolve that merge conflict and fix the image file extension. Feel free to reach out to me if you have questions.

File: about/people.yml (hunk 1 above)
Comment (SimaRaha) [reply about/people.yml:443]: @galenwinsor I already have line 433 in my local code. And the line 443 does not work for me to delete it. So, do I need to push the change to github?

**Review by galenwinsor — APPROVED** [review]

**Comment by eldu** [comment]
> @galenwinsor I'm curious why my bio doesn't appear on the website.

@SimaRaha You need to merge this PR into main in order for it to show up on the actual website. Looks like you'll need to resolve conflicts prior to merging this in

File: about/people.yml [hunk 2]
```diff
@@ -442,6 +442,15 @@ data:
     brown_directory_uuid: 78e02962-a8af-46ec-8636-61237d83cb41
     bio: Michael is a data science intern in the High Performance Computing division at CCV. He is in the Class of 2026 at Brown University studying Computer Science. His academic interests include software development, deep learning, and East Asian studies. Outside of work, you can find him playing basketball or listening to music.
     image: michael1_main.jpg
+  - name: Sima Raha
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Graphics, Software, and Data Core
+    title: Software Sustanibaility Intern
```
Comment (SimaRaha) [inline about/people.yml:449]:
```suggestion
    title: Software Sustainability Intern 
```

**Pull request merged**